src/constants/errors.ts

```diff
@@ -0,0 +1,23 @@
+export const ERRORS = {
+	REJECTED_WALLET: 'Rejected by wallet. Please try again.',
+	DATA_CONSUMED:
+		'The bid you are trying to accept is no longer valid. Please select another bid to accept',
+	PROBLEM_OCCURRED: 'An unexpected problem has occurred. Please try again.',
+	SIGNATURE: 'Failed to generate signature.',
+	TRANSACTION: 'Transaction failed. Please try again',
+	LIBRARY: 'Failed to connect with Web3 wallet.',
+	LIBRARY_NOT_FOUND: 'Could not find web3 library',
+	FAILED_TO_CHECK_ZAUCTION: 'Failed to check zAuction approval status',
+	FAILED_TO_LOAD_DOMAIN_ID: 'Failed to load domain ID',
+	FAILED_TO_RETRIEVE_BID_DATA: 'Failed to retrieve bid data',
+	FAILED_TO_GET_DATA: 'Failed to Get Data',
+	INSUFFICIENT_FUNDS_BID:
+		'You don’t have enough WILD to make that large of a bid.',
```

Comment:
Agreed, I think the only one here I'd expect not to see reused elsewhere in is INSUFFICIENT_FUNDS_BID - I'll move it. The intention here is when we get round to tidying up some of the other flows (removing hard code) we should be able to re-use these constants. It makes sense to re-use the same error copy throughout the dApps, at the moment there are minor variations of the above, just need to go in and replace. I would have done this with the mind set that if its used more than once it can be global const....I think!

Reply:
Done